inc/functions_search.php

```diff
@@ -272,6 +272,8 @@ function clean_keywords_ft($keywords)
 	// Separate braces for further processing
 	$keywords = preg_replace("#((\+|-|<|>|~)?\(|\))#s", " $1 ", $keywords);
 	$keywords = preg_replace("#\s+#s", " ", $keywords);
+	//Replaces less than 3 characters
+	$keywords = preg_replace('/(\b.{1,2}\s)/','',$keywords);
```

Comment:
This should use the setting for "Minimum Search Word Length" (`$mybb->settings['minsearchword']}` and fall back to sensible defaults if it is set to `0` (or a value less than mySQL's stop words default length (4 for full text).